### PR TITLE
baseline first commit

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

**Comment by ehelms** [comment]
Should this include the Pipfile.lock and manifest creation code?

File: system_baseline/config.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# please ensure these are all documented in README.md
+log_level = os.getenv("LOG_LEVEL", "INFO")
+inventory_svc_hostname = os.getenv(
+    "INVENTORY_SVC_URL", "http://inventory_svc_url_is_not_set"
+)
+prometheus_multiproc_dir = os.getenv("prometheus_multiproc_dir", None)
+
+path_prefix = os.getenv("PATH_PREFIX", "/api/")
+app_name = os.getenv("APP_NAME", "system_baselines")
```
Comment (ehelms) [inline system_baseline/config.py]: Should this be plural?

Comment (beav) [reply system_baseline/config.py]: it is unspecified in the doc but others use singular. I'll change it

**Comment by beav** [comment]
> Should this include the Pipfile.lock and manifest creation code?

updated

**Comment by beav** [comment]
@ehelms any further comment, or ok to merge?

**Comment by ehelms** [comment]
ACK

**Pull request merged**